app/models/site_upload.rb

```diff
@@ -69,12 +69,18 @@ def parse_sites
 
       url = Link.normalize(row["url"])
       name = row["nom"] || row["name"]
-      if existing_site = team.sites.find_by_url(url:)
-        existing_site.assign_attributes(tag_ids: tag_ids.union(existing_site.tag_ids))
+      tag_names = row["tags"].present? ? row["tags"].strip.split(",").map(&:strip) : []
```

Comment:
Je pense qu'on peut ne faire qu'un seul strip, et gérer le cas où il y a des virgules en trop.
```suggestion
      tag_names = row["tags"].present? ? row["tags"].split(",").map(&:strip).compact : []
```

Reply:
On a pas besoin du `compact`, `split` et `strip` renverront pas de `nil` 🤔 
À la limite un `compact_blank` oui